wincrypto/Cargo.toml

```diff
@@ -14,4 +14,9 @@ description = "Supporting crate for str0m"
 [dependencies]
 thiserror = { version = "1.0.38" }
 tracing = "0.1.37"
-windows = { version = "0.58", features = ["Win32_Security_Cryptography", "Win32_Security_Authentication_Identity", "Win32_Security_Credentials", "Win32_System_Rpc"] }
+windows = { version = "0.58", features = [
+    "Win32_Security_Cryptography",
+    "Win32_Security_Authentication_Identity",
+    "Win32_Security_Credentials",
+    "Win32_System_Rpc",
+] }
```

Comment:
Was this a deliberate change?

Reply:
Yes, Martin snowflake triggered on this line.

Reply:
Discussed offline will push this